src/profile-logic/profile-data.js

```diff
@@ -147,20 +148,6 @@ export function getSampleCallNodes(
   });
 }
 
-export type SelectedState =
```

Comment:
I decided to move that to `types/profile-derived` because it felt better.

Reply:
Not for this PR: I feel like we need a `types/app.js`, and pull all the non-action types from `types/actions.js` and place them there.

Reply:
yeah we should look at locations for our types and rationalize this a bit.
I'd love to be able to use opaque types too, but I think this would bring some complexity...